### PR TITLE
Backport/tao 10574/login template

### PR DESCRIPTION
**Pull request by SergiiTao**
Ticket: https://oat-sa.atlassian.net/browse/TAO-10574

Backport of https://oat-sa.atlassian.net/browse/TAO-10503

### CODE REVIEW

File: helpers/Template.php [hunk 1]
```diff
@@ -71,29 +72,30 @@ public static function js($path, $extensionId = null) {
...
         }
 
-        if(count($data) > 0){
+        if (!empty($data)) {
             \RenderContext::pushContext($data);
         }
-        
-        $absPath = self::getTemplate($path, $extensionId);
+
+        $absPath = Layout::getThemeTemplate(Theme::CONTEXT_BACKOFFICE, $path) ?? self::getTemplate($path, $extensionId);
```
Comment (ktsirkou) [inline helpers/Template.php:97]: What is the PHP version running on DEPP env?

**Review by ktsirkou — CHANGES_REQUESTED** [review]

File: helpers/Template.php (hunk 1 above)
Comment (SergiiTao) [reply helpers/Template.php:97]: good question

Comment (SergiiTao) [reply helpers/Template.php:97]: 7.2 confirmed by Lionel

Comment (ktsirkou) [reply helpers/Template.php:97]: nice

**Review by ktsirkou — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [ ] New code is respecting code style rules
- [ ] New code is respecting best practices
- [ ] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [ ] Acceptance criteria are respected
- [ ] Pull request title and description are meaningful

**Review by wazelin — APPROVED** [review]
- [ ] New code is covered by tests (if applicable)
- [ ] Tests are running successfully (old and new ones) on my local machine (if applicable)
- [x] New code is respecting code style rules
- [x] New code is respecting best practices
- [x] New code is not subject to concurrency issues (if applicable)
- [ ] Feature is working correctly on my local machine (if applicable)
- [x] Acceptance criteria are respected
- [ ] Pull request title and description are meaningful
- [x] Pull request's target is not `master`

**Pull request merged**